metagraph/src/graph/representation/hash/dbg_sshash.cpp

```diff
@@ -0,0 +1,494 @@
+#include "dbg_sshash.hpp"
+
+#include <sdsl/uint256_t.hpp>
+
+#include "common/seq_tools/reverse_complement.hpp"
+#include "common/threads/threading.hpp"
+#include "common/logger.hpp"
+#include "kmer/kmer_extractor.hpp"
+
+
+namespace mtg {
+namespace graph {
+
+const std::string DBGSSHash::alphabet_ = kmer::KmerExtractor2Bit().alphabet;
+
+using Kmer64 = uint64_t;
+using Kmer128 = __uint128_t;
+using Kmer256 = sdsl::uint256_t;
```

Comment:
Do we have some tests that may be used as speed benchmarks for using it?

Reply:
Could be a nice thing to add in the future... I suppose not necessarily as a part of this PR.